zqd/search/worker.go

```diff
@@ -80,8 +64,7 @@ func (w *WorkerOp) Run(ctx context.Context, store storage.Storage, output Output
 
 	switch st := store.(type) {
 	case *archivestore.Storage:
-		return driver.MultiRun(ctx, d, w.proc, zctx, st.StaticSource(w.si), driver.MultiConfig{
-			Span:      w.si.Span,
+		return driver.MultiRun(ctx, d, w.proc, zctx, st.StaticSource(w.ss), driver.MultiConfig{
```

Comment:
This changed too, since WorkerOp.Run no longer requires the store to be passed in 1502